src/extras/BitmapText.js

```diff
@@ -455,6 +455,71 @@ export default class BitmapText extends core.Container
 
         return this._textHeight;
     }
+
+    /**
+     * Register a bitmap font with data and a texture.
+     *
+     * @static
+     * @param {XMLDocument} xml - The XML document data.
+     * @param {PIXI.Texture} texture - Texture with all symbols.
+     * @return {Object} Result font object with font, size, lineHeight and char fields.
+     */
+    static registerFont(xml, texture)
+    {
+        const data = {};
+        const info = xml.getElementsByTagName('info')[0];
+        const common = xml.getElementsByTagName('common')[0];
+
+        data.font = info.getAttribute('face');
+        data.size = parseInt(info.getAttribute('size'), 10);
+        data.lineHeight = parseInt(common.getAttribute('lineHeight'), 10);
+        data.chars = {};
+
+        // parse letters
+        const letters = xml.getElementsByTagName('char');
+
+        for (let i = 0; i < letters.length; i++)
+        {
+            const charCode = parseInt(letters[i].getAttribute('id'), 10);
```

Comment:
Would it be worth caching `letters[i]` here to eke out a slightly faster parsing time?